vulcan/lib/server/workflows/scRNAseq.metadata.json

```diff
@@ -2,7 +2,7 @@
.a. 'set2').\n  3. Comparison between chosen metadata labels as in 2, but focusing among specific cell groups / types. An example use case here would be performing a case vs control comparison within specific cell types. The setup here is similar to configuration 2 except for an additional selection step (step 4). Here the metadata denoting groupings of cells is chosen, followed by indication of which labels of that metadata to target. As an example, to perform a case vs control comparison within all major cell types, after having utilized the Manual Annotations input to group clusters into cell types, one would select this configuration, then select the metadata holding samples' clinical diagnosis. Labels of that diagnosis metadata denoting target cases, perhaps 'Head and Neck Cancer' and 'Kidney Cancer', would then be selected to represent set 1, and labels of that diagnosis metadata which denote target controls, perhaps 'Healthy Control' would then be selected to represent set 2.  Next, the metadata holding cell type annotations, the 'meta' groupings, is selected in 'step 4', as well as which cell types / groups to run a comparison for.  One might select all the cell types at this stage, or just a few.\n- Subsetting: For all configurations, users are provided an option to subset their data prior to calculation. The use cases here are broad, but perhaps the user's goal is to perform a case vs control comparison within all myeloid cells, for which the method would be to use the subsetting option to get rid of all non-myeloid cells, and then use configuration 2 for performing the calculation. Or perhaps the user wants to identify cluster markers while ignoring a cluster that they suspect contains low quality cells, in which case the method would be to select their 'leiden' metadata for subsetting and select to keep all clusters except one, and then utilize configuration 1 for performing the calculation. In any case, to perform subsetting, first check the associated checkbox, then select which metadata to use for subsetting, and then select the labels of that metadata that should be *retained*. Afterwards, the scanpy dataset will be subset to only these cells *prior* to differential expression calculations being performed with the user's chosen configuration.\n- Additional input sets that affect DGE calculation or output:\n  - '7 Post UMAP DGE Calculation Inputs': These inputs are passed along to the function underlying literal differential expression calculations, `scanpy.tl.rank_genes_groups()`.\n  - '8 Post Calculation DGE Cutoffs': These inputs establish cutoffs for the 'filtered' version of the DGE output, which is generated after the 'full' output is constructed.\n\n### Future Feature Addition Plans:\n\n- Additional visualization methods.\n- Back-end updates to improve the user experience by reducing how often analysis steps are re-run.\n\n#### Additional Note: Refer to in-line (?) icons for further information about specific steps/inputs.",
```

Comment:
(nits): I found the following typos:

`For users initerested in adjusting these parameters` should be `For users interested in adjusting these parameters`

`Selecting of which single-cell data` should be `Selecting which single-cell data`?

`performed to idenfiy top markers` should be `performed to identify top markers`

`versus every other cluster in. These` ... seems to be missing something at the end of the sentence? Not sure what the words should be, though...

`Visulaization and Cluster Annotation (optional)` should be `Visualization and Cluster Annotation (optional)`

`the preliminary differntial expression results` should be `the preliminary differential expression results`

`the final signel-cell data object` should be `the final single-cell data object`

`to perforfollow up analysis outside of vulcan.` should be `to perform follow up analysis outside of Vulcan.` ?

(sorry, can't figure out how to do diffs / comments on specific sections of a single line string...)

Reply:
Thanks!  fixed all these